paintBrush.js

```diff
@@ -115,3 +119,35 @@ function clickShow(e){
 
 
 
+
+let bgButton = document.querySelector('.changeBG')
+//
+// button.addEventListener('click' , onclick);
+bgButton.addEventListener('click', backgroundChange)
+
+let bgCount = 0
+
+function backgroundOptions(){
+    if(bgCount === 0){
+        canvas.style.background = 'white'
+    }else if(bgCount === 1){
+        canvas.style.background = 'red'
+    }else if(bgCount === 2){
+        canvas.style.background = 'blue'
+    }else if(bgCount === 3){
+        canvas.style.background = 'green'
+    }else if(bgCount === 4){
+        canvas.style.background = 'yellow'
+    }else if(bgCount === 5){
+        canvas.style.background = 'orange'
+    }
+}
```

Comment:
I don't like this! It has a similar problem to the switch that was used in changing the brush colour - can you think of a similar solution that's more future proof?

The brush colour buttons have a data-color attribute that is used in changing brush, can you figure out a way of using that here as well?